cypress/integration/IntersectionDetector/Visibility_notification.feature

```diff
@@ -0,0 +1,13 @@
+Feature: The IntersectionDetector notifies the consumer when becoming visible
```

Comment:
Maybe we should update the wording here as well, since we changed all the terms from visibility to intersection.

Reply:
Done